pmd-cataloger/build.gradle.kts

```diff
@@ -73,25 +33,120 @@ dependencies {
   testImplementation(files("$buildDir/../../test/test-jars/apex/testjar-categories-and-rulesets-1.jar"))
 }
 
+
+// ======== MODIFY PLUGIN PROPERTIES ===================================================================================
 java.sourceCompatibility = JavaVersion.VERSION_1_8
 
 application {
   mainClass.set("sfdc.sfdx.scanner.pmd.Main");
 }
 
-// Running the cli locally needs the dist exploded, so just do that
-// automatically with build for ease of use.
+jacoco {
+  toolVersion = "0.8.7"
+}
+
+val distDir = "$buildDir/../../dist"
+
+
+// ======== DEFINE/UPDATE PMD-CATALOGER DIST RELATED TASKS =============================================================
+val pmdCatalogerDistDir = "$distDir/pmd-cataloger"
+
 tasks.named<Sync>("installDist") {
-  into("$distDir/pmd-cataloger")
+  // The installDist task comes with the distribution plugin which comes with the applciation plugin. We modify it here:
+  into(pmdCatalogerDistDir)
+}
+
+tasks.register<Delete>("deletePmdCatalogerDist") {
+	delete(pmdCatalogerDistDir)
+}
+
+// ======== DEFINE/UPDATE PMD6 DIST RELATED TASKS  =====================================================================
+val pmd6DistDir = "$distDir/pmd"
+val pmd6Version = "6.55.0"
```

Comment:
Can we keep the version number at the top of the script? This is one of the values we modify every release (when PMD is actively updating) and having it at the top makes it easy.